components/generic/CompareImageSlider.vue

```diff
@@ -155,6 +156,18 @@
         this.dragging = false;
       },
 
+      showHideAttribution() {
+        if (this.dragging) {
```

Comment:
Is this required since this method is only called when the user is dragging. Please look at 

```
if (this.dragging) {
          this.sliderPosition = pos / this.imageWidth;
          this.showHideAttribution();
}
```

Reply:
Good suggestion, removed the check here.